pkg/sypgp/sypgp.go

```diff
@@ -670,6 +700,8 @@ func reformatMachineReadableOutput(keyText string) (string, error) {
 	for _, key := range keys {
 		var emailList = ""
 
+		fmt.Println("KEY: ", key)
```

Comment:
Do we really need to print the key here? Or is it some debugging leftover?

Reply:
Yes, this is leftover debugging, it will be removed :+1: